pallets/ethereum-checked/src/benchmarking.rs

```diff
@@ -31,7 +31,7 @@ mod benchmarks {
         let target =
             H160::from_slice(&hex::decode("dfb975d018f03994a3b943808e3aa0964bd78463").unwrap());
         // Calling `store(3)`
-        let input = BoundedVec::<u8, ConstU32<MAX_ETHEREUM_TX_INPUT_SIZE>>::try_from(
+        let input = EthereumTxInput::try_from(
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The code is using `EthereumTxInput::try_from`, but it seems that the `try_from` method is not available for `EthereumTxInput`. Please update the code to use the correct conversion method for `EthereumTxInput`.
```diff
-let input = EthereumTxInput::try_from(
+let input = BoundedVec::<u8, ConstU32<MAX_ETHEREUM_TX_INPUT_SIZE>>::try_from(
     hex::decode("6057361d0000000000000000000000000000000000000000000000000000000000000003")
         .unwrap(),
 )
```

LGTM!


<!-- This is an auto-generated comment by OSS CodeRabbit -->